src/pint/models/timing_model.py

```diff
@@ -1742,12 +1759,58 @@ def scaled_dm_uncertainty(self, toas: TOAs) -> u.Quantity:
             result += nf(toas)
         return result
 
+    def scaled_wideband_uncertainty(self, toas: TOAs) -> np.ndarray:
```

Comment:
Is there a method to split this into the appropriate pieces and return them as Quantities?  I assume so, but maybe that should be in the docstring?